pkg/kubernetes/topology-graph.css

```diff
@@ -39,6 +39,10 @@ kubernetes-topology-graph {
     fill: #9467bd;
 }
 
+.kube-topology g.DeploymentConfig text {
+    font-family: FontAwesome;
```

Comment:
Could you make this light gray? That's the color I used in the openshift web console for this.